content/chainguard/chainguard-images/features/private-apk-repos/index.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Chainguard's Private APK Repositories"
+linktitle: "Private APK Repositories"
+type: "article"
+description: "An overview of how to work with Chainguard's Private APK Repositories."
+date: 2025-02-21T11:07:52+02:00
+lastmod: 2025-02-21T11:07:52+02:00
+draft: false
+tags: ["CONCEPTUAL", "CHAINGUARD IMAGES", "PRODUCT", "REFERENCE"]
+images: []
+menu:
+  docs:
+    parent: "features"
+weight: 028
+toc: true
+---
+
+With Chainguard's Private APK Repositories, you can access packages that are included within your organization's image entitlements. This allows you to build custom images based on components that are already part of your organization catalog.
+
+This guide provides a brief overview of Chainguard's private APK repositories and outlines different ways you can incorporate them into your organization's workflows.
+
+> **NOTE**: Chainguard's private APK repositories feature is currently in the beta phase and it will likely go through changes before becoming generally available.
+
+
+## About Private APK Repositories
+
+Chainguard's private APK repos allow customers to pull secure apk packages from Chainguard. The list of packages available in an organization's private repository is based on the apk repositories that the organization already has access to.
+
+For example, say your organization has access to the [Chainguard MySQL container image](https://images.chainguard.dev/directory/image/mysql/versions). Along with `mysql`, this image comes with other apk packages, including `bash`, `openssl`, and `pwgen`. This means that you'll have access to these apk packages through your organization's private APK repository, along with any others that appear in Chainguard container images that your organization has access to. 
+
+As of this writing, private APK repositories are an opt-in feature. [Contact us](https://www.chainguard.dev/contact?utm=docs) to get started.
+
+## Your Repository Address
+
+Your private APK repository will be available at a URL like the following:
+
+`https://apk.cgr.dev/$ORGANIZATION`
```

Comment:
Can we initialize the variables here, with an example:

```
ORGANIZATION="chainguard.edu"
IMAGE="chainguard-base"
```

That way people can copy / paste throughout?